generators/cloudfoundry/generator.mjs

```diff
@@ -25,7 +25,7 @@ import BaseGenerator from '../base/index.mjs';
 
 import prompts from './prompts.mjs';
 import statistics from '../statistics.cjs';
-import constants from '../generator-constants.cjs';
+import { CLIENT_MAIN_SRC_DIR, SERVER_MAIN_RES_DIR } from '../generator-constants.mjs';
```

Comment:
We will need to use application variables instead of constants in the future for custom dirs.